lib/client.js

```diff
@@ -262,7 +247,40 @@ class AssetComputeClient extends EventEmitter {
     async process(source, renditions, userData) {
         if (!this._registered) {
             await this.register();
+            const waitTime = process.env.REGISTER_WAIT_TIME_MSEC || DEFAULT_REGISTER_WAIT_TIME_MSEC;
```

Comment:
Env var for unit tests? Otherwise I would prefer a passed in option in the constructor.